src/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs

```diff
@@ -85,8 +84,9 @@ private void InvalidInput_Base64Transform(ICryptoTransform transform)
 
             AssertExtensions.Throws<ArgumentNullException>("inputBuffer", () => transform.TransformBlock(null, 0, 0, null, 0));
             AssertExtensions.Throws<ArgumentOutOfRangeException>("inputOffset", () => transform.TransformBlock(Array.Empty<byte>(), -1, 0, null, 0));
-            AssertExtensions.Throws<ArgumentNullException>("dst", () => transform.TransformBlock(data_4bytes, 0, 4, null, 0));
-            AssertExtensions.Throws<ArgumentException>(null, () => transform.TransformBlock(Array.Empty<byte>(), 0, 1, null, 0));
+            AssertExtensions.Throws<ArgumentNullException>("outputBuffer", () => transform.TransformBlock(data_4bytes, 0, transform.InputBlockSize, null, 0));
+            AssertExtensions.Throws<ArgumentException>("inputCount", () => transform.TransformBlock(data_4bytes, 0, transform.InputBlockSize + 1, null, 0));
+            AssertExtensions.Throws<ArgumentException>("inputCount", () => transform.TransformBlock(Array.Empty<byte>(), 0, 1, null, 0));
```

Comment:
Adjustments needed for NetFX compat.

Reply:
Please give me more context about "NetFX compat". I can't follow.